arm-iothub/2017-01-19/swagger/iothub.json

```diff
@@ -0,0 +1,1962 @@
+        "Message": {
+          "description": "The error message.",
+          "type": "string",
+          "readOnly": true
+        },
+        "Details": {
+          "description": "The error details.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubQuotaMetricInfoListResult": {
+      "description": "The JSON-serialized array of IotHubQuotaMetricInfo objects with a next link.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The array of quota metrics objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotHubQuotaMetricInfo"
+          }
+        },
+        "nextLink": {
+          "description": "The next link.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RegistryStatistics": {
+      "description": "Identity registry statistics.",
+      "type": "object",
+      "properties": {
+        "totalDeviceCount": {
+          "format": "int64",
+          "description": "The total count of devices in the identity registry.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "enabledDeviceCount": {
+          "format": "int64",
+          "description": "The count of enabled devices in the identity registry.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "disabledDeviceCount": {
+          "format": "int64",
+          "description": "The count of disabled devices in the identity registry.",
+          "type": "integer",
+          "readOnly": true
+        }
+      }
+    },
+    "JobResponseListResult": {
+      "description": "The JSON-serialized array of JobResponse objects with a next link.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The array of JobResponse objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JobResponse"
+          }
+        },
+        "nextLink": {
+          "description": "The next link.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IotHubSkuDescription": {
+      "description": "SKU properties.",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "sku": {
+          "$ref": "#/definitions/IotHubSkuInfo"
+        },
+        "capacity": {
+          "$ref": "#/definitions/IotHubCapacity"
+        }
+      },
+      "required": [
+        "sku",
+        "capacity"
+      ]
+    },
+    "IotHubCapacity": {
+      "description": "IoT Hub capacity information.",
+      "type": "object",
+      "properties": {
+        "minimum": {
+          "format": "int64",
+          "description": "The minimum number of units.",
+          "type": "integer",
+          "minimum": 1,
```

Comment:
It was a recommendation from @amarzavery to add minimum and maximum wherever there is a clear specific range of values that are possible for integer values.